components/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/internal/dao/DashboardMetadataDAO.java

```diff
@@ -30,46 +30,6 @@
 public interface DashboardMetadataDAO {
 
     /**
-     * Check whether a DashboardMetadata exists with given url
-     *
-     * @param url URL of the DashboardMetadata instance
-     * @return true if a DashboardMetadata exists  with given information, false otherwise.
-     * @throws DashboardException if an error occurs
-     */
-    boolean isExists(String url) throws DashboardException;
```

Comment:
This API is not used and therefore it is unnecessary to keep these in the source. And since this repo is not released yet, it is  better to do cleanup now.